src/main/kotlin/io/github/karmasmp/karma/chat/ChatUtils.kt

```diff
@@ -96,31 +99,22 @@ object ChatUtils {
 object GlobalRenderer : ChatRenderer {
     override fun render(source: Player, sourceDisplayName: Component, message: Component, viewer: Audience): Component {
         val playerHead = Noxesium.buildSkullComponent(source.uniqueId, false, 0, 0, 1.0f)
+        val plainMessage = PlainTextComponentSerializer.plainText().serialize(message)
         if(source.hasPermission("karma.group.admin")) {
-            return if(source.isInStaffMode()) {
-                playerHead
-                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value))
-                    .append(sourceDisplayName.color(NamedTextColor.DARK_RED))
-                    .append(Component.text(" SM", NamedTextColor.GOLD))
-                    .append(Component.text(": "))
-                    .append(message)
-            } else {
-                playerHead
-                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value))
+            return playerHead
+                    .append(Component.text(Formatting.Prefix.ADMIN_PREFIX.value)).append(Component.space())
```

Comment:
Would it not make sense to use minimessage here as well?

Reply:
Done